conf/requirements.txt

```diff
@@ -7,3 +7,4 @@ bs4
 pandas
 requests
 lxml
+json5
```

Comment:
wrong module !!

it shall be `simplejson`

please take care what you are writing here in future !!

Reply:
My bad, actually I was using json5 first to create json file but it was creating some errors. That's why I switched to simplejson